docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.8"
+
+services:
+  theater_api:
+    build:
+      context: ./backend
+      dockerfile: ./build/Dockerfile
+      target: dev
+    image: theater/backend
+    container_name: theater_api
+    command:
+      - --reload
+    ports:
+      - "8015:5000"
+    volumes:
+      - ./backend:/opt/theater/backend
+    environment:
+      UVICORN_PORT: 5000
+      UVICORN_APP: src.main:app
+      UVICORN_WORKERS: 2
+    env_file:
+      - ./backend/.env.local
+    depends_on:
+      theater_db:
+        condition: service_healthy
+
+  theater_migration:
+    build:
+      context: ./backend
+      dockerfile: ./build/Dockerfile
+    image: theater/backend
+    container_name: theater_migration
+    command:
+      - migration
+    volumes:
+      - ./backend:/opt/theater/backend
+    env_file:
+      - ./backend/.env.local
+    depends_on:
+      theater_db:
+        condition: service_healthy
+
+  theater_db:
+    image: postgres:15.3-alpine
+    container_name: theater_db
+    shm_size: 1G
```

Comment:
Why did you choose 1G?

Reply:
I've settled shared memory size just according to my inner feelings :) When deploying to production we can change it or remove entirely, but then the default one will be 64mb